static/js/components/EmploymentTab.js

```diff
@@ -168,7 +168,7 @@ class EmploymentTab extends ProfileTab {
           colored
           onClick={this.addWorkHistoryEntry}
           key="I'm unique!"
-          className={`add-employment-button ${fabSpacing()}`}>
+          className={`profile-add-button ${fabSpacing()}`}>
```

Comment:
do we still need to use `fabSpacing()` here? It looks like you're not using something similar above for the education `<FABButton>`